recline.module

```diff
@@ -433,3 +520,15 @@ function recline_file_download($uri, $field_type = 'file') {
 function recline_file_resup_supported_field_widget_types() {
   return 'recline_widget';
 }
+
+/*
+ * Returns array of possible view types supported by Recline.
+ */
+function recline_view_options() {
```

Comment:
This is being used in several parts like the install. It's just to avoid recreate the array several times in different code places.